Sources/Logger/LoggerManager.swift

```diff
@@ -64,21 +83,55 @@ public class LoggerManager {
         }
     }
 
-    public func logFilesRecords(filteredBy filter: (FileLogEntry) -> Bool = { _ in true }) -> [FileLogEntry]? {
-        loggers
+    /// All logged data from `FileLogger` in a form of `FileLogEntry` array. `FileLogEntry` represents a single well structured log entry from a `FileLogger`.
+    /// NOTE: This function returns any data only if at least one `FileLogger` is registed within the manager.
+    /// - Parameters:
+    ///   - loggerId: ID of a specific `FileLogger` instance if more are registed. If not specified, all log entries from all `FileLogger` instances are to be returned.
+    ///   - filter: Additional filter can be applied to specify what entries are to be returned.
+    public func logFilesRecords(
+      forLoggerWithId loggerId: UUID? = nil,
+      filteredBy filter: (FileLogEntry) -> Bool = { _ in true }
+    ) -> [FileLogEntry]? {
+        (loggerId.flatMap(logger(withId:)).map { [$0] } ?? loggers)
             .compactMap { $0 as? FileLogger }
             .compactMap { $0.logFilesRecords(filteredBy: filter) }
             .flatMap { $0 }
     }
 
-    /// Method to delete all log files if there are any.
-    public func deleteAllLogFilesIfAvailable() {
+
+    /// Delete all log files created by an instance of `FileLogger`.
+    /// NOTE: This function returns any data only if at least one `FileLogger` is registed within the manager.
+    /// - Parameters:
+    ///   - loggerId: ID of a specific `FileLogger` instance if more are registed. If not specified, all files from all `FileLogger` instances are to be deleted.
+    public func deleteAllLogFilesIfAvailable(forLoggerWithId loggerId: UUID? = nil) {
```

Comment:
Vazne chceme na LoggerManageru funkce spoléhající na existenci loggeru konkrétního typu (a k tomu uuid)?

Reply:
Je to divny no, dal sem vsechny takovy funkce pryc z LoggerManageru.